First pass at a CFP post (#1260)
writethedocs/www: docs/conf/prague/2020/news/cfp-open.rst

:template: {{year}}/generic.html

.. post:: June 22, 2020
   :tags: prague-2020, website, cfp, tickets

> Announcing Prague Call for Proposals

Reviewer: ```suggestion
Announcing Prague Call for Proposals and Ticket Sales
```

====================================

> Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/2019/cfp/>`_.

Reviewer 2: This link is to 2019.

Reviewer 2: It should also probably mention the explicit event date:

```suggestion
Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/2020/cfp/>`_ for our Virtual Prague conference, taking place October 18-20, 2020.
```

> Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_.

Reviewer: ```suggestion
Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_ and `ticket sales <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_.
```

> Due to the circumstances that you're all familiar with, we're opening it later than usual, and for a shorter length of time.

Reviewer 2: This feels a bit glib -- we should likely mention explicitly moving it online, at least.

> Due to the global situation that you're all familiar with we've moved the Prague conference online, and changed the dates to 19-20 October 2020.

Reviewer 2: Still not sure why we're talking about it:

```suggestion
Due to the global pandemic we've moved the Prague conference online, and changed the dates to 18-20 October 2020.
```

> This means that we're only leaving our call for proposals open for one month. We know that it hasn't been an easy year. Nevertheless we'd love to see you on our online stage in October, to share your ideas with our inspiring community of documentarians.

Reviewer 2: ```suggestion
This means that we're only leaving our Call for Proposals open for one month. We know that it hasn't been an easy year. Nevertheless we'd love to see you on our online stage in October, to share your ideas with our inspiring community of documentarians.
```

> If there is something you’d really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!

Reviewer: ```suggestion
If there is something you’d really like to see a talk about this year, submit a proposal for it or refer someone else who would be good!
```

Call for Proposals
------------------

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, editor, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

You can read our full `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`__ on the website.
The Call for Proposals will be open until **Midnight {{tz}} on {{cfp.ends}}**.

> 

Reviewer 2: We should mention something about being excited for speakers outside of Europe this year, given the online nature of the event. We should also ask for people to help promote it to folks who might have historically not been able to come.

Tickets
-------

> In the meantime, you can grab a ticket on our `Tickets <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_ page.

Reviewer 2: Tickets aren't on sale currently, right? This would block this announcement on that work.

> We're hoping that folks who would not be able to travel to Prague will be able to join us this time round, both as attendees and as speakers! If you've always wanted to talk about documentation but not been able to travel for whatever reason, now is the time.

Reviewer 2: ❤️

As with the previous years, we have tiered pricing to make it fair for everyone. Please buy the ticket that best fits your situation.

> As always, we keep ticket prices low so that it is accessible to the widest range of people possible. However, if you can't afford these prices and still wish to attend, please email us at prague@writethedocs.org, and we can see how we can help.

Reviewer 2: Are we not having a grants program? We should mention the grants program here, not ask folks to email us.

> We'll be opening ticket sales very soon, in the meantime, you can take a look at the new prices for the online conference on our `Tickets <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_ page.

Reviewer: ```suggestion
Tickets are now on sale! Take a look at the new prices for the online conference on our `Tickets <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_ page.

If you previously purchased a ticket to our Portland conference and chose to convert your ticket to a virtual conference pass, you'll receive an email from us in the next weeks to confirm your registration to the Prague virtual conference.
```

> Grant tickets

Reviewer: ```suggestion
Opportunity grants
```

-------------

> We'll be opening our grant program very soon.

Reviewer 2: I don't think we need a title for this, we should just introduce the grant program in the above paragraph. We can likely copy from the Portland introduction, explaining a bit about why we're doing it.

> Stay Updated

Reviewer: ```suggestion
Stay updated
```

Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third-parties, and you can unsubscribe at any time.

Want to connect with other documentarians in real-time? Join our `Slack <http://slack.writethedocs.org/>`_.

> Conclusion

Reviewer 2: This feels a bit weak -- I's probably remove the heading and just close it out in the main body text.
